app/services/theme.js

```diff
@@ -22,6 +22,14 @@ export default Ember.Service.extend({
     // The id of the current provider
     id: config.Reviews.defaultProvider,
 
+    provider: Ember.computed('id', function() {
```

Comment:
Parts of ember-osf will break without `theme.provider` and `theme.isProvider`. We need to have it (because that's less work than fixing ember-osf), but probably shouldn't use it for now.